LICENSE

```diff
@@ -0,0 +1,661 @@
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
## Please change the LICENSE to MIT/equivalent license.

I believe the contents of this repository should be available for free to all, individual, small business, or enterprise, regardless of their purpose with it. Their business models and its ethical nature are of minimal concern to us. The information we want to spread is free, and shall remain so.

Reply:
Done